reference/tools/sbom.rst

```diff
@@ -0,0 +1,92 @@
+.. _conan_tools_sbom:
+
+.. include:: ../../common/experimental_warning.inc
+
+conan.tools.sbom
+=================
+
+A Software Bill of Materials (SBOM) is a document that lists all the components, libraries,
+dependencies, and other elements that make up a specific piece of software. Similar to a bill
+of materials in manufacturing, which details the parts and materials used to build a product,
+an SBOM provides transparency about what is contained "inside" an application or software system.
+
+Conan allows you to generate SBOMs natively by using the resolved dependency graph.
+This way, you can create the SBOM for your program at the same time you build it.
+
+For now, **this feature is in an experimental state, which means that the interface, functionality or generated
+files may change in the future**. Additionally, it currently only supports CycloneDX version 1.4.
+If you need a different standard, another version, or if you encounter any potential improvements,
+please feel free to open an issue on our `GitHub <https://github.com/conan-io/conan/issues>`_ .
+We would be delighted to hear your feedback!
+
+CycloneDX
+^^^^^^^^^
+
+CycloneDX is one of the most widely used standards for SBOMs, supported by the OWASP Foundation.
+
+Using this feature is as simple as implementing a :ref:`hook <reference_extensions_hooks>` in your client,
+which uses this tool to create the SBOM and simply stores it in the appropriate location.
+
+Let's look at two examples:
+
+In the first one, we have the case where we want to generate the SBOM at the moment we create our app, after the
+package method. This is very useful for keeping track of the components and dependencies of our software.
+In the example, we save it in the metadata folder to keep our project organized.
+
+.. code-block:: python
+
+    import json
+    import os
+    from conan.api.output import ConanOutput
+    from conan.tools.sbom.cyclonedx import cyclonedx_1_4
+    def post_package(conanfile):
```

Comment:
```suggestion
    def post_package(conanfile, **kwargs):
```

Most hook examples should have this so that in the future we can more easily update their signature if it's ever needed